cpp/doxygen/header.html

```diff
@@ -0,0 +1,62 @@
+<!-- HTML header for doxygen 1.8.20-->
```

Comment:
Sorry I'm a bit late to the review process. The content of this file doesn't seem specific to cuspatial (and other RAPIDS library). Any reason this cannot be included in rapidsai/docs?

I thought about auto generating this file in the sphinx's build process and automate the injection of the RAPIDS actions and stylesheets. But perhaps a bit brittle if doxygen updates their generate process.

Reply:
> Sorry I'm a bit late to the review process. The content of this file doesn't seem specific to cuspatial (and other RAPIDS library). Any reason this cannot be included in rapidsai/docs?

That's unfortunately not possible because the doxygen builds happen at the repository level and not in `rapidsai/docs`.